### PR TITLE
Commerce 4263 user invitation by email

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
this pr was ported from https://github.com/marco-leo/liferay-portal-ee/pull/1187

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/batch/v1_0/AccountMemberBatchEngineTaskItemDelegate.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+@Component(
+	enabled = false, immediate = true,
+	service = BatchEngineTaskItemDelegate.class
+)
+public class AccountMemberBatchEngineTaskItemDelegate
+	extends BaseBatchEngineTaskItemDelegate<AccountMember> {
+
+	@Override
+	public void createItem(
+			AccountMember item, Map<String, Serializable> parameters)
```
[the remark this answers is not in the document]
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/batch/v1_0/AccountMemberBatchEngineTaskItemDelegate.java]: force pushed changes request

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
rebased

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
@marco-leo @riccardo-alberti I've asked the #commerce-usm-collaboration and they told me that no changes are needed on this Pull in order to met Account Port criteria so I've just put this ticket back on the review queue

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**